implementations/micrometer-registry-prometheus/src/main/java/io/micrometer/prometheus/MicrometerCollector.java

```diff
@@ -77,6 +78,37 @@ public boolean isEmpty() {
                 .collect(toList());
     }
 
+    @Override
+    public List<MetricFamilySamples> describe() {
+        switch (id.getType()) {
+            case COUNTER:
+                return Collections.singletonList(
+                        new MetricFamilySamples(conventionName, Type.COUNTER, help, Collections.emptyList()));
+
+            case GAUGE:
+                return Collections.singletonList(
+                        new MetricFamilySamples(conventionName, Type.GAUGE, help, Collections.emptyList()));
+
+            case TIMER:
+                return Collections.singletonList(
+                        new MetricFamilySamples(conventionName, Type.HISTOGRAM, help, Collections.emptyList()));
```

Comment:
Also need the max GAUGE like the distribution summary

Reply:
We should increase test cases for filtering by time series name to ensure we're not missing any like this. For example, with a Timer named `my.timer` and attempting to filter by `my_timer_duration_seconds_max` (`PrometheusMeterRegistryTest` uses the `PrometheusDurationNamingConvention` by default) does not work as is right now.

Reply:
@shakuzen Thanks for pointing it out! I changed as suggested.